src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md

```diff
@@ -3,7 +3,7 @@ title: Getting Started with Ethereum Development Using Alchemy
 description: "This is a beginners guide to getting started with Ethereum development using Alchemy, We’ll take you from signing up with Alchemy to making a command line request, to writing your first web3 script! No blockchain development experience necessary!"
 author: "Elan Halpern"
 tags:
-  ["Getting started", "web3.js", "Ethers.js", "Nodes", "Querying", "Alchemy"]
+  ["Getting started", "javascript", "Ethers.js", "Nodes", "Querying", "Alchemy"]
```

Comment:
Noticing that the case of our tags are creating duplicates, e.g. `Querying` in this post vs `querying` elsewhere:
<img width="907" alt="Image 2020-12-01 at 8 30 20 AM" src="https://user-images.githubusercontent.com/8097623/100757368-2518cc00-33a3-11eb-91bd-0525272a84cd.png">
We should establish a standard for this. Any thoughts? Title case? Sentence case?